src/main/java/com/microsoft/graph/HttpResponseCode.java

```diff
@@ -0,0 +1,44 @@
+package com.microsoft.graph;
+
+public final class  HttpResponseCode {
+
+    /**
+     * HTTP response code OK
+     */
+    public static final int HTTP_OK = 200;
```

Comment:
Out of curiosity, is there any chance these can be pulled from one of the http/java libraries we depend on?